boot/zephyr/pm.yml

```diff
@@ -85,3 +85,9 @@ mcuboot_secondary_1:
   size: CONFIG_NRF53_RAM_FLASH_SIZE
 
 #endif /* CONFIG_NRF53_MULTI_IMAGE_UPDATE */
+
+#if (CONFIG_MCUBOOT_HW_DOWNGRADE_PREVENTION)
+provision:
+  size: CONFIG_PM_PARTITION_SIZE_PROVISION
+  region: otp
+#endif /* CONFIG_MCUBOOT_HW_DOWNGRADE_PREVENTION */
```

Comment:
Hi, why is this needed?

I would think that when

CONFIG_SECURE_BOOT_STORAGE is enabled this code would already be added though this mechanism:

https://github.com/nrfconnect/sdk-nrf/pull/9196/files#diff-f434425bbc55a28763ba74d09be74c00824ba6d3f768af979614867b3485ec54R81

Perhaps the Kconfig is disabled?

Shall we introduce a new Kconfig that adds pm.yml.secure_boot_storage to the build?

Or should the Kconfig be enabled?

Reply:
The reason that we need this is that MCUBOOT counters can exist without bl_storage. If we don't use the NSIB but we still want to use the hardware counters for MCUBOOT the partition will not be generated since the SECURE_BOOT_STORAGE is not generated.